style.css

```diff
@@ -0,0 +1,488 @@
+/* ElementX font */
+@import url('https://fonts.googleapis.com/css2?family=Orbitron:wght@700&display=swap');
+
+/* Font for index */
+@import url('https://fonts.googleapis.com/css2?family=Hind&display=swap');
+
+/* Font for Content */
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@300;400;700&display=swap');
+
+:root {
```

Comment:
Since the root file will be imported as a whole in the web, there might be conflicting names with the website that is using your lib as it has very common names like of colors. I faced the same issue in the library so! I would suggest renaming all the color schemes with something like: `--ele-X-colorname` so that there are minimal chances of conflict.

Reply:
Done